### ISSUE TITLE
min_cycle_recoil only affects black powder cartridges

### ISSUE THREAD

**Issue by Outta-my-way-burrocrat** [CleverRaven/Cataclysm-DDA#34338]
# Describe the bug

Changing the American-180's min_cycle_recoil doesn't affect its ability to fire CB ammunition.

# Steps To Reproduce

Part 1:
1. Increase the American-180's min_cycle_recoil to 100 
2. Load it with .22 CB cartridges.
3. Switch to automatic mode.
4. Shoot a practice target.

Part 2:
1. Increase the American-180's min_cycle_recoil to 500 
2. Load it with .22 LR cartridges.
3. Switch to automatic mode.
4. Shoot a practice target.

# Expected behavior

Failure to cycle.

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.D-8025-g6139406
 - Graphics version: Tiles
 - Mods loaded: Dark Days Ahead; Disable NPC Needs

**Comment by tenmillimaster** [comment]
It's not coded to handle this, but this would be a good use of it.

**Comment by anoobindisguise** [comment]
I've seen a Saiga-12 get jammed by firing beanbag rounds, so I think this can be closed?

**Comment by Outta-my-way-burrocrat** [comment]
Nope, regular ammunition spawned from the debug menu is still unaffected by min_cycle_recoil.

**Comment by Outta-my-way-burrocrat** [comment]
Still an issue but I'm closing out my account so....

**Comment by NetSysFire** [comment]
No reason to close a valid issue.

**Comment by RenechCDDA** [comment]
This does appear to be fully resolved. I went on a bit of wild goose chase after editing the wrong json value (ugh!) but it all works as expected when I set the min_cycle_recoil above the round's threshold.

That json failure on my part also caused me to pore over the code and keep asking "This should work, why doesn't it work". Spoiler alert: It does work, but again, wrong json value I edited.

@NetSysFire Should be good to close unless you reproduced this?
